src/app/modules/submission/submission.utils.ts

```diff
@@ -1,11 +1,34 @@
-import { keyBy } from 'lodash'
+import _, { keyBy } from 'lodash'
```

Comment:
for consistency, should we exclusively either use `_` or the destructured function? I'm leaning towards destructured but you can choose :P